pkg/apis/core/v1alpha2/types.go

```diff
@@ -110,6 +112,23 @@ type GroupSpec struct {
 	IPBlock *secv1a1.IPBlock `json:"ipBlock,omitempty"`
 }
 
+// GroupPhase defines the phase in which a Group is.
+type GroupPhase string
+
+// These are the valid values for GroupPhase.
+const (
+	// GroupPending means the Group has been accepted by the system, but it has not been processed by Antrea.
+	GroupPending GroupPhase = "Pending"
+	// GroupRealized means the Group has realized all of its GroupMembers.
+	GroupRealized GroupPhase = "Realized"
+)
+
+// GroupStatus represents information about the status of a Group.
+type GroupStatus struct {
+	// The phase of a Group is a simple, high-level summary of the Group's status.
+	Phase GroupPhase `json:"phase"`
```

Comment:
@tnqn I noticed you also have ObservedGeneration to store the generation of the object in NetworkPolicyStatus. I intentionally left it out from here because the consumers would only be interested in whether the CG is realized or not and does not really perform an action based on change in state. So used only for reporting purpose. Let me know of your thoughts on this

Reply:
Sure, I don't think ObservedGeneration is must to have if no use case of it.